help/search/_posts/2021-05-23-search-parser.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title: "(Advanced) Search Syntax"
```

Comment:
it's governed by `aqp.maxAbsolutePhraseLength` but this parameter is invariant. I do not allow users to change it (it can only be changed in solrconfig.xml) because that would be a safety hazard

Reply:
Ok, but why don't we say that this (pasting long phrases of text and having the search engine find relevant records) is a feature of our system?  It is a useful way to search for similar text but it's undocumented.